packages/presentation/pubspec.yaml

```diff
@@ -15,12 +15,19 @@ dependencies:
 
   data:
     path: ../data
+  domain:
+    path: ../domain
```

Comment:
no need in domain dependency here. data already has domain

Reply:
done